backend/tests/core/Entities/EntityHelper.cs

```diff
@@ -0,0 +1,24 @@
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Core.Test
+{
+    /// <summary>
+    /// EntityHelper static class, provides helper methods to create test entities.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public static partial class EntityHelper
+    {
+        /// <summary>
+        /// Generate a random rowversion.
+        /// </summary>
+        /// <param name="data">Data to convert into row version.</param>
+        /// <returns></returns>
+        public static byte[] GenerateRowVersion(string data)
```

Comment:
Simple way to generate a row version instead of using the same one everywhere.  This is useful for testing the templates as they use the rowversion to ensure the compiled template is unique.